### ISSUE TITLE
Installability Testing

### ISSUE THREAD

**Issue by smiths** [smiths/vdisp#104]
Test vdisp to ensure that it is easy to install on Mac, Windows and Linux machines.  The number of steps required should be small, and the user expertise shouldn't be assumed to be technical.

Before starting testing instructions should be written for users.  (The current instructions are more aimed at developers.)

The steps to testing installability are as follows:

1. Informal testing with friends and family.  (@EmilSoleymani, let me know when you are ready and I'll get my son to install our software on his laptop.)
2. Send the instructions to @dstolle to ask for his feedback.
3. Send the instructions to people outside of our group, like Dr. Gabriel Sedran and possibly the current instructors for geotechnical courses at Mac.  For this test we should do usability testing at the same time.  I'll create a separate issue for that.
4. Record the installability testing results in the test report document.  (We don't currently have a test report document, but we should create one.  We also need a test (verification and validation) plan.)

**Comment by smiths** [comment]
@EmilSoleymani I've installed and run the `update-output` branch version of `vdisp` and I have the following observations for the instructions:
- we should warn users that `make precompile` will take a while
- we should warn users that the first time running the software will take longer than usual
